src/jdk.hotspot.agent/share/man/jhsdb.1

```diff
@@ -1,4 +1,4 @@
-.\" Copyright (c) 2019, 2021, Oracle and/or its affiliates. All rights reserved.
+.\" Copyright (c) 2019, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This seems not correct?

Reply:
According to the comments in the makefile (`closed/make/UpdateOpenManPages.gmk`) the copyright line is taken from the original Markdown file, so if the year is wrong there, it will be wrong in the generated nroff file.

I think it would be incorrect to edit the dates locally in these files, because they'll just be overwritten when we generate the files again. Ideally, the dates should be fixed (if necessary) in the Markdown files, but that seems out of scope for this P1.

This is "just" an issue with copyright dates in source files ... and yes, while I know copyright dates are important, this problem is arguably part of an ongoing more general problem.

I note that the generated files *do* correctly identify themselves with `2021` in the visible output generated to the console by the `man` command.

Reply:
Thanks for the explanation, Jon. Fine by me.

Reply:
I agree that fixing this date is not necessary at this time.